{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/ekaterina-epetrova/Conference-Info/blob/master/ASFV_proposal.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "7fa798ed-705d-4366-8402-07f81dfe711a",
      "metadata": {
        "id": "7fa798ed-705d-4366-8402-07f81dfe711a"
      },
      "source": [
        "# African swine fever\n",
        "\n",
        "African swine fever virus (ASFV) is a virus that poses a serious threat to agriculture as the etiological agent of the hemorrhagic African swine fever (ASF). Twenty four distinct genotypes have been reported based on the sequence of B646L. No current trerapeutic approach is currently available.\n",
        "\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Correlation of Neutralizing Antibody Levels with Vaccine Effectiveness\n",
        "\n",
        "The correlation between neutralizing antibody levels and vaccine effectiveness is strong, especially in the early period following vaccination. High levels of these antibodies are generally associated with a robust ability to prevent infection or to reduce the severity of the disease. However, as time progresses from the point of vaccination, several studies have shown that these antibody levels can decrease. This decrease does not linearly correlate with a drop in vaccine effectiveness due to several reasons:\n",
        "\n",
        "## Memory B and T Cells\n",
        "\n",
        "The immune response to vaccination isn't solely dependent on neutralizing antibodies. Memory B cells and T cells also play a critical role in providing protection. Even if antibody levels wane, these memory cells can quickly spring into action upon re-exposure to the pathogen, producing new antibodies and attacking infected cells, respectively.\n",
        "\n",
        "## Breadth of Response\n",
        "\n",
        "The immune system's response to a vaccine is broad, targeting multiple parts of the virus. Even if antibodies against one part of the virus decrease, immunity isn't entirely lost as other components of the immune response can still offer protection.\n",
        "\n",
        "## Threshold for Protection\n",
        "\n",
        "The exact level of antibodies required to confer protection can vary, and for some diseases, even relatively low levels of antibodies can be protective. Additionally, the correlation between neutralizing antibody titers and protection can be nonlinear\n"
      ],
      "metadata": {
        "id": "F9LIK2KsTZo9"
      },
      "id": "F9LIK2KsTZo9"
    },
    {
      "cell_type": "markdown",
      "id": "befba989-dbed-4dca-b300-0ba0a1117e93",
      "metadata": {
        "id": "befba989-dbed-4dca-b300-0ba0a1117e93"
      },
      "source": [
        "Proposal: Profiling B Cell Repertoire in Pigs Infected with African Swine Fever Virus (ASFV)\n",
        "\n",
        "1. Introduction:\n",
        "African Swine Fever (ASF) is a viral disease affecting domestic pigs and wild boars, leading to significant economic losses in the swine industry. Understanding the host immune response, particularly the B cell repertoire, during ASFV infection is crucial for developing effective diagnostic and therapeutic strategies. Profiling the B cell repertoire can provide insights into the diversity and dynamics of antibody responses against ASFV, aiding in the identification of vaccine targets.\n",
        "\n",
        "2. Objectives:\n",
        "\n",
        "2.1. To characterize the B cell repertoire in pigs infected with attenuated ASFV\n",
        "2.2. To identify clonal expansions and diversity dynamics within the B cell population during ASFV infection\n",
        "2.3. To correlate changes in the B cell repertoire with disease progression and outcomes.\n",
        "2.4. To clone sequence for m\n",
        "\n",
        "3. Methodology:\n",
        "a. Sample Collection: Blood samples will be collected from ASFV-infected pigs at different time points post-infection (e.g., pre-infection, acute phase, convalescent phase).\n",
        "A very high viral titer can be observed in various lymphatic organs, especially spleen, as demonstrated before.\n",
        "\n",
        "b. B Cell Isolation: Peripheral blood mononuclear cells (PBMCs) will be isolated using density gradient centrifugation, followed by isolation of B cells using magnetic bead-based sorting.\n",
        "\n",
        "c. BCR Sequencing: Total RNA will be extracted from sorted B cells, and B cell receptor (BCR) sequences will be amplified using multiplex PCR targeting variable (V) and joining (J) gene segments. High-throughput sequencing will be performed to obtain BCR repertoire data.\n",
        "\n",
        "d. Bioinformatic Analysis: BCR sequences will be processed to identify clonal expansions, determine CDR3 lengths, and calculate diversity metrics. Phylogenetic analysis will be conducted using metrics such as Levenshtein distance to infer evolutionary relationships among BCRs.\n",
        "\n",
        "e. Functional Relevance Analysis: Integration of BCR sequencing data with functional assays, such as antibody binding assays or virus neutralization assays, will be performed to assess the functional relevance of identified BCRs and antibodies.\n",
        "\n",
        "4. Expected Outcomes:\n",
        "\n",
        "Comprehensive profiling of the B cell repertoire dynamics in pigs infected with ASFV.\n",
        "Identification of clonal expansions and diversity patterns associated with ASFV infection.\n",
        "Correlation between changes in the B cell repertoire and disease progression.\n",
        "Discovery of potentially novel antibodies with functional relevance against ASFV.\n",
        "Insights into the immune response mechanisms underlying ASFV infection.\n",
        "5. Significance and Impact:\n",
        "\n",
        "The proposed study will advance our understanding of the B cell response to ASFV infection, providing valuable insights into host-pathogen interactions.\n",
        "The identified BCRs and antibodies may serve as candidates for diagnostic biomarkers or therapeutic targets for controlling ASF.\n",
        "Findings from this study will contribute to the development of effective vaccines and immunotherapies against ASFV.\n",
        "6. Future Directions:\n",
        "\n",
        "Validation of identified BCRs and antibodies through functional assays and in vivo studies.\n",
        "Investigation of the long-term durability and memory response of B cells following ASFV infection.\n",
        "Exploration of potential strategies to modulate the B cell response for enhancing ASFV immunity.\n",
        "In conclusion, the proposed study aims to elucidate the dynamics and functional relevance of the B cell repertoire in pigs infected with ASFV, with implications for both understanding disease pathogenesis and developing interventions against ASF."
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Useful resources:\n",
        "\n",
        "\n",
        "## Antibodies:\n",
        "\n",
        "* [Immunological_toolbox](https://www.immunologicaltoolbox.co.uk/)\n",
        "\n",
        "* https://www.ars.usda.gov/northeast-area/beltsville-md-bhnrc/beltsville-human-nutrition-research-center/diet-genomics-and-immunology-laboratory/docs/dgil-porcine-translational-research-database/\n",
        "\n"
      ],
      "metadata": {
        "id": "Ed-mTYPfzwnu"
      },
      "id": "Ed-mTYPfzwnu"
    },
    {
      "cell_type": "markdown",
      "id": "3f559972-924c-4317-8f27-e32c7801b626",
      "metadata": {
        "id": "3f559972-924c-4317-8f27-e32c7801b626"
      },
      "source": [
        "## B Cell Isolation and available markers:\n",
        "\n",
        "Since pan B cell marker is missing the isolation of the B cells is not trivial. We can apply negative gating as well as negative selection of antigen reactive B cells against capsid proteins such as p72 and p30\n",
        "\n",
        "\n",
        "\n",
        "### **Markers**\n",
        "\n",
        "#### Detection of porcine plasma cells\n",
        "| Marker | Conjugate | Clone | Isotype | Source | Dilution | Function |\n",
        "|--------|-----------|-------|---------|--------|----------|----------|\n",
        "| CD49d | BV605 | L25 | mouse IgG2b | BD Biosciences | 1:33 | binding fibronectin|\n",
        "| CD79α | PerCP-Cy5.5 | HM47 | mouse IgG1 | BioLegend | 1:200 |\n",
        "| IRF4 | PE | 3E4 | rat IgG1 | Thermo Fisher | 1:40 |\n",
        "| IL10  | AlexaFluor647 | 945A 4C4 37B1| mouse IgG1 | Thermo Fisher | 0.6 μL in 10 μL |\n",
        "\n",
        "**Cells were fixed and permeabilised with:** eBioscience transcription factor staining Buffer Set (Thermo Fisher, Cat: 00-5523-00).\n",
        "\n",
        "Reference: [Gerner, 2022](https://www.sciencedirect.com/science/article/pii/S0145305X22001501)\n",
        "\n",
        "\n",
        "### Sample:\n",
        "\n",
        "#### B cell distribution\n",
        "* PBMCs- ~2% of the total B cell pool\n",
        "* Lymph nodes- 28%\n",
        "* Spleen and mucosal surface- 23%\n",
        "* BM- 17%\n",
        "\n",
        "#### Time:\n",
        "* before vaccination\n",
        "* after vaccination- timing is importnat\n",
        "* after challenge\n",
        "\n",
        "### Type of sequencing:\n",
        "\n",
        "#### Bulk sequencing\n",
        "\n",
        "Bulk seq can provide information for clonality, isotype frequency, V gene usage. However, it is difficult to provide information for paired Ig repetoire thus reconstruction of the Ab is tricky. The only option is frequency analysis as descibed by Reddy et al., 2020\n",
        "\n",
        "1. gDNA- clonality is better defined\n",
        "2. mRNA\n",
        "\n",
        "#### sc sequencing:\n",
        "1. scBCR-seq\n",
        "2. scRNA-seq\n",
        "\n",
        "\n",
        "\n",
        "* Sequencing depth: sequencing depth must be greater than the number of B cells in the sample\n",
        "How many B cells are of interest?"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Objectives:\n",
        "\n",
        "1. Identify IgGVH and IgGVL genes whose product possess \"intact ASFV\" specific binding potential\n",
        "Note: apoptotic cells might be releasing non-enveloped ASFV that can raise naked ASFV antibodies\n",
        "2. Relative abundance of V-gene transcripts in naive and ASFV immunize pigs\n",
        "3. Analyse the dynamics of the adaptive immune response in ASFV vaccinated  \n",
        "\n",
        "## Approaches to investigate antibody repertoire:\n",
        "\n",
        "### Type of sequencing:\n",
        "* bulk RNAseq- cheap\n",
        "* scRNAseq- individual IgGH and IgHL as well as in parallel investigation of gene expression levels of essential genes\n",
        "\n",
        "### Analysis platform\n",
        "* network theory (Miho et al, 2019)\n",
        "* biological diversity indexes (Briney et al, 2019)\n",
        "* statistical analysis of germlines and complementarity determining region 3 (CDR3) (Reddy et al., 2010)\n",
        "* mathematical modeling of VDJ recombination of antibodies (Elhanati, 2015)\n",
        "\n",
        "\n",
        "### Workflow:\n",
        "1. Analyse the antibody repertoire during the vaccination with ASFV-E and upon challenge with ASFV-G (bulk)\n",
        "- dynamics of antibody response and identify specific lineage\n",
        "- altered gene expression in bulk\n",
        "2. Analyse the antibody repertoire and gene expression level in single cell resolution\n",
        "-genes with altered expression\n",
        "3. Where are the plasma cells? Tonsils, bone marrow\n",
        "\n",
        "\n",
        "## Experimental design:\n",
        "\n",
        "*   number or animals\n",
        "\n",
        "How many animals?\n",
        "\n",
        "*   type of animals- survivors and non-survivors after the ASFV-G challenge\n",
        "\n",
        "- Is there some difference during vaccination and upon challenge?\n",
        "- Is there difference between the groups\n",
        "\n",
        "*   number of time points- 0 dpi, 7dpi (p72 seropositive), ..., post challenge\n",
        "\n",
        "- Can we identify stereotypic Abs?\n",
        "Indication of the B cell diffrentiation from mature B cells to plasma cells can be derived by the presence of IgM/IgG transition in the blood\n",
        "\n",
        "## Source of material and type of B cells?\n",
        "\n",
        "What tissue do you currently have availble? Artur mentioned that the BM is not working?\n",
        "\n",
        "  * Bone marrow- long-lived plasma B cells but it is the same place for pigs maybe\n",
        "The long-lived plasma B cells represent < 1% lymphoid cells. In mice, after 6 dpi there is an enrichment of BMPCs that persist for prolong periods.\n",
        "  * spleen- short-lived plasma cells\n",
        "In mice, adter 6 dpi there is a enrichment of plasma cells which concentration rapidly decline after 11 dpi.\n",
        "\n",
        "  * blood- probably during transitioning between the LN and bone marrow\n",
        "\n",
        "\n",
        "## Type of IgG-seq\n",
        "  * gDNA\n",
        "  * mRNA\n",
        "\n",
        "__To do:__\n",
        "*   design degenerate V gene primer mixes\n",
        "\n",
        "\n",
        "## Approach:\n",
        "\n",
        "\n",
        "**1.   Statistical approach (Reddy et al. 2010)**\n",
        "\n",
        "~5000 V gene seq is ok for antibody discovery\n",
        "\n",
        "**Overview:**\n",
        "*   B cells isolation:\n",
        "    * limited number of surface markers for detection of plasma B cells\n",
        "    * CD3-CD16-CD172a-CD49-/+ (Villanueva-Hernandez et al., 2022)\n",
        "*   extraction of mRNA and reverse transcription in bulk\n",
        "*   PCR amplification with degenerative V-gene primer mixes\n",
        "    * need to design and order\n",
        "* sequencing of the amplicons\n",
        "  * What is the average length in order to decide what sequencing platform to be used\n",
        "* bioinformatics:\n",
        "\n"
      ],
      "metadata": {
        "id": "irYXc7BfpRGz"
      },
      "id": "irYXc7BfpRGz"
    }
  ],
  "metadata": {
    "kernelspec": {
      "display_name": "Python 3 (ipykernel)",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.9.13"
    },
    "colab": {
      "provenance": [],
      "toc_visible": true,
      "include_colab_link": true
    }
  },
  "nbformat": 4,
  "nbformat_minor": 5
}